components/FooterSliceZone.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <footer class="footer" role="contentinfo">
+    <div class="footer__left">
+      <div
+        v-for="(slice, index) in usedMainFooter.data.body"
+        :key="`footer-slice-left-${index}`"
+      >
+        <template v-if="slice.slice_type === 'logos_zone'">
+          <logos-zone :slice="slice" class="footer-left__logos" />
+        </template>
+        <prismic-rich-text
+          v-if="slice.slice_type === 'text'"
+          class="footer-left__description"
+          :field="slice.primary.text"
+        />
+        <div
+          v-if="slice.slice_type === 'social_media'"
+          class="footer-left__social-media"
+        >
+          <pix-link
+            v-for="(sm, smIndex) in slice.items"
+            :key="`sm-${smIndex}`"
+            :field="sm.socialmedia_url"
+            class="footer-social-media__icon"
+          >
```

Comment:
**suggestion (non-blocking):** Pour réduire la complexité cognitive, il est préférable d'éviter l'utilisation d'abréviation et ainsi d'être plus explicite dans le nommage

```suggestion
          <pix-link
            v-for="(socialMedia, index) in slice.items"
            :key="`sm-${index}`"
            :field="socialMedia.socialmedia_url"
            class="footer-social-media__icon"
          >
```

Reply:
Le `index` doit porter un autre nom car on est dans un autre v-for avec déjà une `index`

Reply:
`socialMediaIndex` ?